app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,9 @@
     <meta name='viewport' content='width=device-width, initial-scale=1.0'>
     <%= render partial: 'layouts/favicon' %>
     <link rel='manifest' href='manifest.json'>
-    <%= render partial: 'layouts/google_analytics' %>
+    <% if ENV['GOOGLE_ANALYTICS_ID'].present? %>
```

Comment:
Mostly because it sort of hid the fact that the partial may not render anything at all. But if the majority of the people agree I can move it back there.

Reply:
@janklimo @hiattp @roonglit So, what's everyone's preference on this? Keep the conditional in the partial so the layout stays cleaner or be more explicit about the conditions under which th Google Analytics code will be added?

Reply:
@citizen428 I lean towards keeping them in the partial. More in line with _tell, don't ask_ IMO.